Net07.HW/Net07.Operators/Net07.Operators/Program.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+
+namespace Net07.Operators
+{
+    class Program
+    {
+        enum Currency
+        {
+            EUR,
+            USD,
+            RUB,
+            BYN
+        }
+
+        const double EUR = 3.1762d;
+        const double USD = 2.6293d;
+        const double RUB = 0.0346d;
+        const double BYN = 1.0000d;
+        const double PERCENTTMARKUP = 1.002d;
+        static double markUpEUR = Math.Round(EUR * PERCENTTMARKUP, 4);
+        static double markUpUSD = Math.Round(USD * PERCENTTMARKUP, 4);
+        static double markUpRUB = Math.Round(RUB * PERCENTTMARKUP, 4);
+        const double FI = 1.6180339887d;
+        const string patternBinary = @"(?i)(^\d{1,})\s{0,}(pow|\+|\-|\/|\*|\%)\s{0,}(\d{1,}$)";
```

Comment:
`{1,}` can be replaced by `+` and {0,} can be replaced by `*`
`+` means 1 or more `*` means 0 or more